parser/src/java/org/apache/hadoop/hive/ql/parse/FromClauseParser.g

```diff
@@ -306,7 +316,7 @@ valuesTableConstructor
 @init { gParent.pushMsg("values table constructor", state); }
 @after { gParent.popMsg(state); }
     :
-    valueRowConstructor (COMMA! valueRowConstructor)*
+    valueRowConstructor (options{greedy=true;}: COMMA! valueRowConstructor)*
```

Comment:
Can you leave a comment about the `greedy` option behavior?

Reply:
Removed explicit set of `greedy` option since tests are passed without it.